### PR TITLE
Reestructure

### LINKED ISSUE #33: Namespace everything under ActiveAdminChat module
I think there are some classes/modules in ActiveAdminChat that are not correctly namespaced with a `module ActiveAdminChat` . Check it and correct it if needed

### PR DESCRIPTION
**Pull request by santib**
Resolves #33 

Goals:
1. Namespace all the gem code
2. Follow Rubygems suggestions for gem naming

### CODE REVIEW

File: app/helpers/active_admin/chat/chat_helper.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module ActiveAdmin
+  module Chat
+    module ChatHelper
+      def admin_class(message)
+        'admin' if message.sender.class == ActiveAdmin::Chat.admin_user_klass
```
Comment (juanmanuelramallo) [inline app/helpers/active_admin/chat/chat_helper.rb:5]: What do you think of `is_a?` ?
```suggestion
        'admin' if message.sender.is_a?(ActiveAdmin::Chat.admin_user_klass)
```

Comment (santib) [reply app/helpers/active_admin/chat/chat_helper.rb:5]: I like it, but let's do that as part of a separate PR

**Review by juanmanuelramallo — APPROVED** [review]
W H O A

**Review by fedeagripa — APPROVED** [review]
Sunday = cleaning day 💪

**Review by f19ps — APPROVED** [review]
🥇

**Review by jpascual1994 — APPROVED** [review]
🚀

**Pull request merged**